.github/workflows/release.yaml

```
@@ -84,39 +97,65 @@ jobs:
           CMAKE_APPLE_SILICON_PROCESSOR=arm64 \
           ./hack/static.sh all
 
-          mkdir -p ./libgit2-darwin-libgit2-all/lib
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/include ./libgit2-darwin-libgit2-all/
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/share ./libgit2-darwin-libgit2-all/
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/cmake ./libgit2-darwin-libgit2-all/lib/
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/engines-3 ./libgit2-darwin-libgit2-all/lib/
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/ossl-modules ./libgit2-darwin-libgit2-all/lib/
-          mv ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/pkgconfig ./libgit2-darwin-libgit2-all/lib/
 
-          libtool -static -o ./libgit2-darwin-libgit2-all/lib/libcrypto.a \
-            ${GITHUB_WORKSPACE}/build/libgit2-darwin-amd64/lib/libcrypto.a \
+          LIBGIT2_SED="s;-L/Applications/Xcode_.* ;;g"
```

Shouldn't this be?
```suggestion
          LIBGIT2_SED="s;-L/Applications/Xcode.* ;;g"
```

Also, we should be doing this when we are building libgit2 only as well? This is what `libgit2.pc` looks like on my mac (running macOS 12) when I build libgit2 only:
```
prefix="/Users/sanskarjaiswal/Development/fluxcd/golang-with-libgit2/build/libgit2"
libdir="/Users/sanskarjaiswal/Development/fluxcd/golang-with-libgit2/build/libgit2/lib"
includedir="/Users/sanskarjaiswal/Development/fluxcd/golang-with-libgit2/build/libgit2/include"
                                                                                                                                   
Name: libgit2
Description: The git library, take 2
Version: 1.3.1
Libs: -L${libdir} -lgit2
Libs.private: -L/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/usr/lib -liconv
Cflags: -I${includedir}
```

---

This was extracted from the script we currently run at both IAC and SC and is based on the current runner's OS:
https://github.com/fluxcd/source-controller/blob/main/hack/install-libraries.sh#L59

I updated it so it will continue to work when we upgrade the runner OS to macOS 12.

---

Leaving this comment unresolved, in case doubts arise on this in the future.